appveyor.yml

```diff
@@ -18,8 +18,8 @@ cache:
 - C:\ProgramData\chocolatey\bin -> appveyor.yml
 - C:\ProgramData\chocolatey\lib -> appveyor.yml
 - '%USERPROFILE%\.nuget\packages -> **\*.csproj'
-- '%LOCALAPPDATA%\npm-cache -> **\package.json'          # npm cache
-- '**\node_modules -> **\package.json'              # local npm modules
+- '%APPDATA%\npm-cache -> **\package.json'          # npm cache
+- '..\node_modules -> **\package.json'              # local npm modules
```

Comment:
@jpknoll The ** to .. seems to have fixed the problem. Please review when you get a minute.